### PR TITLE
Removed ALBA logos as default

### LINKED ISSUE #67: Remove ALBA logos as default
Calipsoplus is an EU project with contributions from many facilities and users. The ALBA logos should not be set as default. 

Of course each facility should have the option to configure the logos to their facility on a local version.

### PR DESCRIPTION
**Pull request by aicampbell**
close #67

### CODE REVIEW

File: src/app/calipso-constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // put the logo facitility url
-export const LOGO_FACILITY = 'https://www.cells.es/logo.png';
+export const LOGO_FACILITY = 'assets/images/insert-logo-here.jpg';
```
Comment (dsanchez-cells) [inline src/app/calipso-constants.ts]: Minor thing, but maybe this should be just "logo.jpg" or read from environment settings?

Comment (aicampbell) [reply src/app/calipso-constants.ts]: I was just using the code already provided.
But yes, this should be in the environments instead of a random file.

Comment (dsanchez-cells) [reply src/app/calipso-constants.ts]: Better to load it from environment.ts then, say "environment.logo".

Comment (aicampbell) [reply src/app/calipso-constants.ts]: I'm doing it now

**Review by dsanchez-cells — APPROVED** [review]

**Comment by dsanchez-cells** [comment]
Hi Aidan, sorry for not noticing earlier, but the example files should also be updated. After that and if Travis allows, we can proceed with the merge.

**Comment by aicampbell** [comment]
It was my fault, I'm fixing the missing file and test file

**Pull request merged**